modules/ROOT/pages/salesforce/salesforce-connector-100-xml-maven-edit.adoc

```diff
@@ -0,0 +1,70 @@
+= Salesforce XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule app in XML either from the Studio XML editor or a text editor, perform the following actions to get access to the connector:
+
+Add the namespace for the connector to your XML flow.
+Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the canvas, Studio automatically populates the XML code with the connector namespace and schema location.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/salesforce
+http://www.mulesoft.org/schema/mule/salesforce/current/mule-salesforce.xsd
+----
+
+This example shows how the namespace statements are placed in the mule XML block:
+
+[source,xml,linenums]
+----
+<mule xmlns:salesforce="http://www.mulesoft.org/schema/mule/salesforce"
+	xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:file="http://www.mulesoft.org/schema/mule/file"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/file
+	http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
+	http://www.mulesoft.org/schema/mule/http
+	http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/ee/core
+	http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+	http://www.mulesoft.org/schema/mule/salesforce
+	http://www.mulesoft.org/schema/mule/salesforce/current/mule-salesforce.xsd">
+----
+
+== Add a POM File Dependency
+
+The `pom.xml` Maven file generated by Anypoint Studio contains dependencies for a Mule app. When you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector.
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule-salesforce-connector</artifactId>
+  <version>10.0.0</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+To obtain the most up-to-date `pom.xml` file information, access the connector in
+https://www.mulesoft.com/exchange/[Anypoint Exchange], and click *Dependency Snippets*.
+
+== Next
+
+After completing your namespace and `pom.xml` file, you can try out the xref:salesforce/salesforce-connector-examples.adoc[Salesforce Examples].
```

Comment:
I often wonder about "try out" to a nonnative speaker. Maybe just "try"? "test"? "sample"?

Reply:
Changed to "try creating."